packages/amplify_core/lib/src/config/auth/cognito/oauth.dart

```diff
@@ -27,6 +29,46 @@ class CognitoOAuthConfig
     this.tokenUriQueryParameters,
   });
 
+  @internal
+  factory CognitoOAuthConfig.fromAuthOutputs(AuthOutputs authOutputs) {
```

Comment:
P.S. this used in `AuthConfiguration.fromAmplifyOutputs` AuthConfiguration is an internal type used in Auth State Machine and is differnt from AuthConfig.